drgn_tools/task.py

```diff
@@ -197,6 +197,91 @@ def get_command(task: Object) -> str:
     return escape_ascii_string(task.comm.string_())
 
 
+def get_ppid(task: Object) -> int:
+    """
+    :returns: Parent PID of the task
+    """
+    return task.parent.pid.value_()
+
+
+def get_rss(prog: Program, task: Object) -> int:
+    """
+    :returns RSS value
+    """
+    try:
+        filerss = task.mm.rss_stat.count[0].counter.value_()
+        if filerss < 0:
+            filerss = 0
+        anonrss = task.mm.rss_stat.count[1].counter.value_()
+        if anonrss < 0:
+            anonrss = 0
+        shmrss = task.mm.rss_stat.count[3].counter.value_()
+        if shmrss < 0:
+            shmrss = 0
+        rss_mm = filerss + anonrss
+
+    except Exception:
+        rss_mm = 0
```

Comment:
`shmrss` is unused... why collect it? same below as well.

Reply:
This is done to inline with Oracle crash tool. As discussed, we will make it similar to
main line Crash , which is more appropriate

Reply:
Got it, thanks!